src/models/tools/drawing/drawing-objects.ts

```diff
@@ -0,0 +1,64 @@
+export interface Point {x: number; y: number; }
+export interface DeltaPoint {dx: number; dy: number; }
+
+export interface BoundingBox {
+  nw: Point;
+  se: Point;
+  start?: Point;
+  end?: Point;
+}
+
+export interface LineDrawingObjectData {
+  [key: string]: any;
```

Comment:
These were to fix a typescript error when setting properties on the drawing objects

```
if (drawingObject && drawingObject.model.hasOwnProperty(prop)) {
  drawingObject.model[prop] = newValue;
}
```
TS complained that i couldn't do this because type 'DrawingObjectDataType' has no index signature.

However, removing the arbitrary additional properties allowed me to spot two small bugs, and in any case I realize that offers more protection to remove them and just to cast the above line as `any`.

:+1: